variables.tf

```diff
@@ -172,5 +177,5 @@ variable "capacity_provider_strategies" {
 variable "ephemeral_storage_gib" {
   description = "The total amount, in GiB, of ephemeral storage to set for the task. The minimum supported value is 20 GiB and the maximum supported value is 200 GiB."
   type        = number
-  default     = 20
```

Comment:
wut

Reply:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_task_definition#ephemeral_storage

🤷🏽

Reply:
@dmattia Updated the variable's description, and need an approval again!